src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const semicolon = ';';
+  const colon = ':';
```

Comment:
Creating a variable for semicolon and colon  and using it only once is unnecessary and complicates the code without providing any benefits.
remove it